src/reducers/collections.js

```diff
@@ -38,6 +38,12 @@ function validateCollection(configCollection) {
     // Cannot infer format from extension.
     throw new Error(`Please set a format for collection "${ collectionName }". Supported formats are ${ supportedFormats.join(',') }`);
   }
+  const frontmatterFormats = ['yaml-frontmatter','toml-frontmatter','json-frontmatter']
```

Comment:
I'd rather have this list imported from `src/formats/formats.js`, like we did for the `supportedFormats` one above. Just easier to keep it together.

Reply:
Fixed. I will think more carefully about where I place things from now, thanks for your patience.

Reply:
Oh no, not a problem at all. Sorry to bug you with little things.